src/dispatch/incident_cost/models.py

```diff
@@ -0,0 +1,41 @@
+from sqlalchemy import Column, ForeignKey, Integer, Numeric
+from sqlalchemy.orm import relationship
+
+from dispatch.database import Base
+from dispatch.incident_cost_type.models import (
+    IncidentCostTypeCreate,
+    IncidentCostTypeUpdate,
+    IncidentCostTypeRead,
+)
+from dispatch.models import DispatchBase, TimeStampMixin
+
+
+# SQLAlchemy Model
+class IncidentCost(Base, TimeStampMixin):
+    # columns
+    id = Column(Integer, primary_key=True)
+    amount = Column(Numeric(precision=10, scale=2), nullable=False)
```

Comment:
We could consider something like a currency type here:

https://sqlalchemy-utils.readthedocs.io/en/latest/_modules/sqlalchemy_utils/types/currency.html

In case we want to support other types of currency.

Reply:
My idea was to keep it generic here and then add a setting under /configuration/general, or something like that, where we'd allow users to specify the currency. Then display the amount according to that setting. We could also move other settings in the `.env` to /configuration/general.